pkg/kubelet/metrics/metrics.go

```diff
@@ -31,30 +31,42 @@ import (
 )
 
 const (
-	KubeletSubsystem             = "kubelet"
-	NodeNameKey                  = "node_name"
-	NodeLabelKey                 = "node"
-	PodWorkerLatencyKey          = "pod_worker_latency_microseconds"
-	PodStartLatencyKey           = "pod_start_latency_microseconds"
-	CgroupManagerOperationsKey   = "cgroup_manager_latency_microseconds"
-	PodWorkerStartLatencyKey     = "pod_worker_start_latency_microseconds"
-	PLEGRelistLatencyKey         = "pleg_relist_latency_microseconds"
-	PLEGDiscardEventsKey         = "pleg_discard_events"
-	PLEGRelistIntervalKey        = "pleg_relist_interval_microseconds"
-	EvictionStatsAgeKey          = "eviction_stats_age_microseconds"
-	VolumeStatsCapacityBytesKey  = "volume_stats_capacity_bytes"
-	VolumeStatsAvailableBytesKey = "volume_stats_available_bytes"
-	VolumeStatsUsedBytesKey      = "volume_stats_used_bytes"
-	VolumeStatsInodesKey         = "volume_stats_inodes"
-	VolumeStatsInodesFreeKey     = "volume_stats_inodes_free"
-	VolumeStatsInodesUsedKey     = "volume_stats_inodes_used"
+	KubeletSubsystem                     = "kubelet"
+	NodeNameKey                          = "node_name"
+	NodeLabelKey                         = "node"
+	PodWorkerLatencyKey                  = "pod_worker_latency_seconds"
```

Comment:
I just noticed this on another PR, these should all be `duration` instead of `latency`

Reply:
Does this looks good ? https://github.com/kubernetes/kubernetes/pull/72470/commits/79a3eb816c66d9928e02b8589a05fd0de2b8f378

Reply:
So we should do this change to all other metrics ?

Reply:
Yes we should.

Reply:
sure, I will change them.